portfolio/src/main/webapp/statistics.html

```diff
@@ -67,38 +67,38 @@ <h3 class="section-subheading text-muted">
           <div class="dropdown">
             <button
               type="button"
-              class="btn btn-primary dropdown-toggle"
+              class="btn btn-secondary dropdown-toggle"
               data-toggle="dropdown"
               aria-haspopup="true"
               aria-expanded="false"
             >
               Where martyrs have died
```

Comment:
Not all martyrs we have featured died, for example Justin Howell was just severely injured. Maybe think about changing the name displayed on the button.

Reply:
I think i'm going to remove Justin Howell from the list to keep the weight and significance of the list